rwengine/src/engine/Garage.cpp

```diff
@@ -168,12 +168,18 @@ bool Garage::isObjectInsideGarage(GameObject* object) const {
     auto p = object->getPosition();
 
     // Do basic check first
-    if (p.x < min.x) return false;
-    if (p.y < min.y) return false;
-    if (p.z < min.z) return false;
-    if (p.x > max.x) return false;
-    if (p.y > max.y) return false;
-    if (p.z > max.z) return false;
+    if (p.x < min.x)
```

Comment:
nit: won't one big `if` look better?

Reply:
I actually like the current code.
I setted `AllowShortBlocksOnASingleLine` to false (in `.clang-format`), maybe set it to true?